src/entity/User.ts

```diff
@@ -135,7 +149,7 @@ const checkUsernameAndEmail = async (
     .getRepository(User)
     .createQueryBuilder()
     .select('id')
-    .where('email = :email or username = :username', {
```

Comment:
I think we should do a lowercase while saving the user (if we are not doing that already)